src/state/reducers/windows.js

```diff
@@ -90,10 +90,6 @@ export const windowsReducer = (state = {}, action) => {
           y: action.payload.size.y,
         },
       };
-    case ActionTypes.NEXT_CANVAS:
```

Comment:
This was dead code, yeah? I wonder if we should put this in a separate PR for clarity?

Reply:
Yes, there is no reducer handling this action anymore. I think it was replaced by `SET_CANVAS`. I'll extract this and put it into a separate pr.